patches/chrome-utility-importer-safari_importer.mm.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/utility/importer/safari_importer.mm b/chrome/utility/importer/safari_importer.mm
+index a83d19200f9b73f3adb22db0005891ab80d3dc80..7943f958997afe14390df7ceb77e95ade3385b5d 100644
+--- a/chrome/utility/importer/safari_importer.mm
++++ b/chrome/utility/importer/safari_importer.mm
+@@ -47,6 +47,7 @@ void SafariImporter::StartImport(const importer::SourceProfile& source_profile,
+     ImportBookmarks();
+     bridge_->NotifyItemEnded(importer::FAVORITES);
+   }
++  BRAVE_START_IMPORT
```

Comment:
why are we doing this? Is it really necessary to have behavior for Safari import that Chrome doesn't?

Reply:
this is not worth a patch imo

Reply:
cc @simonhong

Reply:
Chrome's importing feature on macOS was broken for a long time because chrome can't access that data w/o full disk access. So far, we support this by subclassing that method and notifying users about this needs full disk access guide dialog.
Chrome team inallyremoved their broken implementation and this patch is introduced.

Reply:
@simonhong We can hold off on this one.

Reply:
Ok, Importing history from safari is removed from C87.